data-model/fdo-profiles/0.1.0/mapping/examples/example-mapping-request.json

```diff
@@ -0,0 +1,11 @@
+{
+  "data": {
+    "type": "https://hdl.handle.net/21.T11148/ce794a6f4df42eb7e77e",
+    "attributes": {
+      "pidIssuer":"https://ror.org/04wxnsj81",
+      "issuedForAgent":"https://ror.org/0566bfb96",
+      "structuralType":"physical",
+      "sourceDataStandard": "dwca"
```

Comment:
This should be `dwc`. Also noticed that this is not an enum in the schemas. Might be something we want to change? Let get it consistent here and then see if we need a task to make it an enum.